Unchumped wonder closet.

Allspice opened the pull request:
16 sets. Dubious balance. SUGGESTIONS STILL WELCOME.

:cl:
* tweak: Add some worthwhile stuff to the wonder closet maybe perhaps

otatoh commented:
More ideas: dracula coat, hos/warden/officer alt jackets, darkholme suit (also useful to falseflag traitor items).

ShiftyRail commented:
If the wonder closet has a guaranteed chance of gettting the HoS longcoat (which is RNG-based on main station and has 50% of spawning, 50% of getting a duplicate of the normal trenchcoat), then traders are my best friends

MadmanMartian commented on code/game/objects/items/trader.dm:
```diff
@@ -58,8 +58,24 @@
+	list(/obj/item/clothing/suit/space/clown, /obj/item/clothing/head/helmet/space/clown),
+	/obj/item/clothing/shoes/magboots/magnificent,
+	list(/obj/item/clothing/suit/space/plasmaman/bee, /obj/item/clothing/head/helmet/space/plasmaman/bee),
+	list(/obj/item/clothing/head/celtic, /obj/item/clothing/suit/armor/ice),
```
Celtic crown's supposed to be high-tier unique vault loot.

AI-Is-Rogue commented:
How about the lich crown and lich robe/tattered robe.

Dacendeth commented:
How about all the winter coats?

DamianX commented:
Please look into the suggestions posted here

YertBogan commented:
Who among the player base even knows all of the items that are worth including? The whole fun of the wardrobe was that you didn't know what you were going to get. How do I suggest what I don't know exists?

MadmanMartian commented:
every vox trader, as all 2 of them frequently hunt for vaults

Allspice commented:
I removed the celt stuff and added the lichwear, sure. Darkholme's already in the theatre vend, could add Sundowner but that can already spawn for the HOS and might make the closet too armour-heavy, and vamp basically ``sucks dick".
